hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/procedures/RunBootstrapProcedure.scala

```diff
@@ -85,6 +85,7 @@ class RunBootstrapProcedure extends BaseProcedure with ProcedureBuilder with Log
     val propsFilePath = getArgValueOrDefault(args, PARAMETERS(15)).get.asInstanceOf[String]
     val bootstrapOverwrite = getArgValueOrDefault(args, PARAMETERS(16)).get.asInstanceOf[Boolean]
 
+    val (tableName, database) = HoodieCLIUtils.getHoodieDatabaseAndTable(table.get.asInstanceOf[String])
     val configs: util.List[String] = new util.ArrayList[String]
```

Comment:
Should the database name be a separate param?

Reply:
Yes, it would be nice if the database name was a separate parameter. However, other existing procedures allow the table name to contain the database name, so I choose to keep this PR change consistent with the existing logic.
For example:
https://github.com/apache/hudi/blob/master/hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieCLIUtils.scala#L62